### PR TITLE
Add results of WMT 2015-2021.md

### PR DESCRIPTION
**Pull request by liashahnazaryan**
Update wmt21.md to add results of the shared news translation task.

### CODE REVIEW

File: events/wmt21.md [hunk 1]
```diff
@@ -209,3 +209,32 @@ Thursday, November 11, 2021
...
+## Results
+
+### Shared Task: Machine Translation of News
+
+#### Translation into-English (SR+DC)
+*Systems ordered by DA score z-score; systems within a cluster are considered tied; lines indicate clusters according to Wilcoxon rank-sum test p < 0.05*
+
+|     |     |     |     |     |
+| --- | --- | --- | --- | --- |
+| **Language pair** | **Rank** | **Ave.** | **Ave. z** | **System** |
```
Comment (bittlingmayer) [inline events/wmt21.md:222]: Put this in the header line and remove the `**`, it will then bold automatically.

```
| Language pair | Rank | Average | Average z-score | System |
| --- | --- | --- | --- | --- |
| Czech → English | 1 – 2 | 77.8 | 0.111 | `Facebook-AI` |
...
```

Please remember that everything needs to be machine-translated one day, and things like *Ave. z* and *Borderline* will be machine-translated terribly.  So expand abbreviations and use backticks for untranslatables accordingly.

File: events/wmt21.md [hunk 2]
```diff
@@ -209,3 +209,32 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+### Shared Task: Machine Translation of News
```
Comment (bittlingmayer) [inline events/wmt21.md:215]: > ## Results
> Results of the shared tasks
> 
> - [*Findings of the 2021 Conference on Machine Translation (WMT21)*](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
> ### News translation
>
> #### → English

File: events/wmt21.md [hunk 3]
```diff
@@ -209,3 +209,32 @@ Thursday, November 11, 2021
...
+
+#### Translation into-English (SR+DC)
```
Comment (bittlingmayer) [inline events/wmt21.md:217]: Is this a direct quote?  If so, use `<`.

File: events/wmt21.md (hunk 1 above)
Comment (bittlingmayer) [inline events/wmt21.md:222]: What does Rank mean here?

File: events/wmt21.md (hunk 3 above)
Comment (bittlingmayer) [reply events/wmt21.md:217]: If it is, it's fine.

If it's not, then we should try to explain it more simply.

File: events/wmt21.md [hunk 4]
```diff
@@ -209,3 +209,32 @@ Thursday, November 11, 2021
...
+#### Translation out-of-English
+*Systems ordered by DA score z-score; systems within a cluster are considered tied; lines indicate clusters according to Wilcoxon rank-sum test p < 0.05*
+
+|     |     |     |     |     |
+| --- | --- | --- | --- | --- |
+
+
+
+
+[Summary of the results](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
```
Comment (bittlingmayer) [inline events/wmt21.md:240]: This is a hardly summary.  What *you* did is a summary.

We can just put it at the top.

Or say here:

> - Full results: [*Findings of the 2021 Conference on Machine Translation (WMT21)*](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)

File: events/wmt21.md [hunk 5]
```diff
@@ -209,3 +209,32 @@ Thursday, November 11, 2021
...
+| **Language pair** | **Rank** | **Ave.** | **Ave. z** | **System** |
+| Czech→English | 1–2 | 77.8 | 0.111 | Facebook-AI |
+| German→English | 1–5 | 71.9 | 0.126 | Borderline |
+| Hausa→English | 1 | 74.4 | 0.248 | Facebook-AI |
+| Icelandic→English | 1 | 74.5 | 0.293 | Facebook-AI |
+| Japanese→English | 1 | 73.8 | 0.141 | HW-TSC |
+| Russian→English | 1–5 | 77.5 | 0.137 | NVIDIA-NeMo |
+| Chinese→English | 1–5 | 75.0 | 0.042 | NiuTrans |
+
+#### Translation out-of-English
```
Comment (bittlingmayer) [inline events/wmt21.md:231]: > #### English →

File: events/wmt21.md (hunk 1 above)
Comment (liashahnazaryan) [reply events/wmt21.md:222]: It's the change in the order of systems, but as I think about it, it may not be valuable without information on clusters, so maybe I should remove it from the table. What do you think?

File: events/wmt21.md [hunk 6]
```diff
@@ -212,29 +212,44 @@ Thursday, November 11, 2021
...
-
-
-
-
-[Summary of the results](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+#### Full results of the shared tasks: [*Findings of the 2021 Conference on Machine Translation (WMT21)*](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+
+### News Translation
+
+#### → English (segment rating with document context (SR+DC))
```
Comment (cefoo) [inline events/wmt21.md:219]: Hi Lia!

Please notice that the Style Guide dictates '[do not use symbols as abbreviations](https://machinetranslate.org/style#acronyms-and-abbreviations)'. This is important for preparing the website to be machine translated in the future.

File: events/wmt21.md [hunk 7]
```diff
@@ -212,29 +212,44 @@ Thursday, November 11, 2021
...
+| --- | --- | --- | --- |
+| Czech→English | 77.8 | 0.111 | `Facebook-AI` |
+| German→English | 71.9 | 0.126 | `Borderline` |
+| Hausa→English | 74.4 | 0.248 | `Facebook-AI` |
+| Icelandic→English | 74.5 | 0.293 | `Facebook-AI` |
+| Japanese→English | 73.8 | 0.141 | `HW-TSC` |
+| Russian→English | 77.5 | 0.137 | `NVIDIA-NeMo` |
+| Chinese→English | 75.0 | 0.042 | `NiuTrans`|
+
+#### English → (segment ranking in a full document (SR+FD))
```
Comment (cefoo) [inline events/wmt21.md:232]: Same as above

File: events/wmt21.md [hunk 8]
```diff
@@ -209,3 +209,47 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+#### Full results of the shared tasks: [*Findings of the 2021 Conference on Machine Translation (WMT21)*](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
```
Comment (cefoo) [inline events/wmt21.md:215]: Hi dear @liashahnazaryan!

I've just realized that we have followed this approach when talking about conferences:

- When a talk refers to a published paper, its mention is formatted in italics and bold.  For instance, in [WMT21](https://machinetranslate.org/wmt21#day-1).
- When there is no published paper about it, its name is just bold, as in [MTM 2019](https://machinetranslate.org/mtm2019).

Thanks!

Comment (liashahnazaryan) [reply events/wmt21.md:215]: hey, @cefoo!
I've made changes accordingly.

File: events/wmt21.md [hunk 9]
```diff
@@ -217,7 +217,7 @@ Thursday, November 11, 2021
 ### News Translation
 
 #### → English (segment rating with document context)
-> *Systems ordered by Direct Assessment score z-score; systems within a cluster are considered tied; lines indicate clusters according to Wilcoxon rank-sum test p < 0.05>*
```
Comment (liashahnazaryan) [inline events/wmt21.md]: dear @cefoo, I've also deleted some parts of the description as I've just realized that those are not relevant if we show only winners and not all the results

File: events/wmt21.md [hunk 10]
```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
...
+| French→German | 87.7 | 0.088 | `Online-W`|
+| German→French | 87.9 | 0.160 | `Online-B`|
+
+#### French to/from German (segment rating in a full document)
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| French→German | 87.7 | 0.088 | `Online-W`|
+| German→French | 87.9 | 0.160 | `Online-B`|
+
```
Comment (liashahnazaryan) [inline events/wmt21.md:251]: several changes to kept consistency with the structure of yearly results in wmt articles

File: events/wmt21.md [hunk 11]
```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
...
+#### Full results of the shared tasks: [***Findings of the 2021 Conference on Machine Translation (WMT21)***](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+
+### News Translation
+
+#### → English (segment rating with document context)
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| Czech→English | 77.8 | 0.111 | `Facebook-AI` |
```
Comment (bittlingmayer) [inline events/wmt21.md:223]: I'd put a space around the arrow.

Maybe we can remove "English" since it's in the section title?

File: events/wmt21.md [hunk 12]
```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+#### Full results of the shared tasks: [***Findings of the 2021 Conference on Machine Translation (WMT21)***](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
```
Comment (bittlingmayer) [inline events/wmt21.md:215]: I'd make this just normal text or a bullet point, not a title.  And maybe just italics, not bold and italics?

> - Full results of the shared tasks: [*Findings of the 2021 Conference on Machine Translation (WMT21)*](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)

File: events/wmt21.md [hunk 13]
```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
...
+
+### News Translation
+
+#### → English (segment rating with document context)
```
Comment (bittlingmayer) [inline events/wmt21.md:219]: I would move the parenthetical out of the title.  It messes up anchor links etc.

Can make a sentence like

> The human evaluation was based on **segment ratings with document context**.

File: events/wmt21.md [hunk 14]
```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
...
+| German→French | 87.9 | 0.160 | `Online-B`|
+
+#### French ↔︎ German (segment rating in a full document)
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| French→German | 87.7 | 0.088 | `Online-W`|
+| German→French | 87.9 | 0.160 | `Online-B`|
+
+#### Hindi ↔︎ Bengali and Zulu ↔︎ Xhosa (segment rating without document context)
```
Comment (bittlingmayer) [inline events/wmt21.md:252]: > #### Hindi ↔︎ Bengali
> ...
>
> #### Zulu ↔︎ Xhosa
> ...

File: events/wmt21.md [hunk 15]
```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+#### Full results of the shared tasks: [***Findings of the 2021 Conference on Machine Translation (WMT21)***](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+
+### News Translation
```
Comment (bittlingmayer) [inline events/wmt21.md:217]: > ### News translation

**Review by liashahnazaryan — COMMENTED** [review]
Made several changes here as well based on your comments, @bittlingmayer

File: events/wmt21.md [hunk 16]
```diff
@@ -209,3 +209,59 @@ Thursday, November 11, 2021
...
+| Japanese → | 73.8 | 0.141 | `HW-TSC` |
+| Russian → | 77.5 | 0.137 | `NVIDIA-NeMo` |
+| Chinese → | 75.0 | 0.042 | `NiuTrans`|
+
+#### English →
+Human evaluation based on segment ratings in a full document.
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| → Czech | 90.2 | 0.397 | `HUMAN-A` |
```
Comment (bittlingmayer) [inline events/wmt21.md:237]: Same as in the other PR

File: events/wmt21.md [hunk 17]
```diff
@@ -209,3 +209,59 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
```
Comment (liashahnazaryan) [inline events/wmt21.md:213]: You can review the changes here as well, @bittlingmayer.

File: events/wmt21.md [hunk 18]
```diff
@@ -209,3 +209,61 @@ Thursday, November 11, 2021
...
+| Hindi → Bengali | `HW-TSC` | 95.0 | 0.245 |
+| Bengali → Hindi | `GTCOM` | 82.1 | 0.202 |
+
+#### Zulu ↔︎ Xhosa
+The human evaluation used direct assessment without document context.
+
+| Language pair | System | Average score | Average z-score |
+| --- | --- | --- | --- |
+| Zulu → Xhosa | `TRANSSION` | 80.7 | 0.502 |
+| Xhosa → Zulu | `HW-TSC` | 68.4 | 0.331 |
```
Comment (liashahnazaryan) [inline events/wmt21.md:269]: Same changes here, @cefoo.

**Review by liashahnazaryan — COMMENTED** [review]
I've added WMT 2015-2021 official results to this pull request, @cefoo, and made several changes according to our discussions.

**Review by liashahnazaryan — COMMENTED** [review]
You can now review the changes, @cefoo! Let me know if there's anything that needs to be changed.

**Review by liashahnazaryan — COMMENTED** [review]
I've updated the links to the term definitions in the results section of each article. They now are directed to the `Evaluation`section of the `WMT` page. Dear @cefoo, please review.

**Comment by cefoo** [comment]
Hi @liashahnazaryan!

I think this looks great! ✨

Just a minor change which I failed to mention (but easy to fix).  According to our [style guide](https://machinetranslate.org/style#spacing), we should separate sentenced with a single enter and paragraphs with two enters.

Could you please check?

Thanks so much!

**Comment by liashahnazaryan** [comment]
> Hi @liashahnazaryan!
> 
> I think this looks great! ✨
> 
> Just a minor change which I failed to mention (but easy to fix). According to our [style guide](https://machinetranslate.org/style#spacing), we should separate sentenced with a single enter and paragraphs with two enters.
> 
> Could you please check?
> 
> Thanks so much!

Thanks for the comment, @cefoo. I fixed it.

**Comment by cefoo** [comment]
Thank you for your hard work and patience, @liashahnazaryan!
